locales/hu-HU.json

```diff
@@ -0,0 +1,335 @@
+{
+    "`x` subscribers": "`x` feliratkozó",
+    "`x` videos": "`x` videó",
+    "`x` playlists": "`x` playlist",
```

Comment:
In general used "playlist" as foreign word is Hungarian as well, kinda "slang", while there is "official" `lejátszási lista` but personally I find it a bit awkward.